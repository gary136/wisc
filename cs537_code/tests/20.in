path tests/p2a-test
loop 3 $loop I love cs537
