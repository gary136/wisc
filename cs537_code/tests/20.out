This is the first argument: I
This is the second argument: love
This is the third argument: cs537
